nanoFramework.WebServer/WebServer.cs

```diff
@@ -468,7 +468,8 @@ public static void SendFileOverHTTP(HttpListenerResponse response, StorageFile s
         /// /// <param name="contentType">The type of file, if empty string, then will use auto detection</param>
```

Comment:
"double comment" here
```suggestion
        /// <param name="contentType">The type of file, if empty string, then will use auto detection</param>
```